flink-streaming-java/src/test/java/org/apache/flink/streaming/api/environment/CheckpointConfigFromConfigurationTest.java

```diff
@@ -166,11 +199,15 @@ public void setNonDefaultValue(CheckpointConfig config) {
 
         public void assertEqual(
                 CheckpointConfig configFromFile, CheckpointConfig configFromSetters) {
-            assertThat(getter.apply(configFromFile), equalTo(getter.apply(configFromSetters)));
+            customAssertion.accept(getter.apply(configFromFile), getter.apply(configFromSetters));
         }
 
         public void assertEqualNonDefault(CheckpointConfig configFromFile) {
-            assertThat(getter.apply(configFromFile), equalTo(nonDefaultValue));
+            customAssertion.accept(getter.apply(configFromFile), nonDefaultValue);
+        }
+
+        public Arguments asArgument() {
```

Comment:
Is this some magical junit5 construct?

Reply:
hmm, still works if it is removed.

Reply:
args, good catch. I added it initially to provide a `Stream<Arguments>` until I realized that it's not necessary for parameter lists of size 1. I forgot to remove it while reverting the change. I updated the branch accordingly